cluster/kube-up.sh

```diff
@@ -53,19 +53,20 @@ echo "... calling kube-up" >&2
 kube-up
 
 echo "... calling validate-cluster" >&2
-if [[ "${EXIT_ON_WEAK_ERROR}" == "true" ]]; then
-	validate-cluster
+# Override errexit
+if validate-cluster; then
```

Comment:
You might be able to get away with `(validate-cluster) && validate_result="$?" || validate_result="$?"`.